src/commands/types.d.ts

```diff
@@ -1,14 +1,13 @@
-import type { NetlifyConfig } from '@netlify/build'
-import type { NetlifyTOML } from '@netlify/build-info'
```

Comment:
These are so, so incomplete and wrong that it isn't worth dealing with them at the moment. This PR tries to get CLI to have its own reliable build types and we'll someday fix them upstream hopefully.